libraries/botframework-connector/tests/auth.test.js

```diff
@@ -424,5 +446,56 @@ describe('Bot Framework Connector - Auth Tests', function() {
                 assert(isValid);
             });
         });
+
+        describe('AuthenticationError', () => {
```

Comment:
Please put these tests in another file... I don't want more tests going into `auth.test.js`

Reply:
moved to `authenticationError.test.js`